.travis.yml

```diff
@@ -4,8 +4,7 @@ language: groovy
 
 jdk:
   - oraclejdk8

```

Comment:
I recommend not using Java 8 but 7. The reason being that we should always cater for the lowest common denominator. People using this plugin might not necessarily be using Java 8, which will cause their builds to break.